sale_by_packaging/readme/CONTRIBUTORS.rst

```diff
@@ -1,3 +1,5 @@
 * Akim Juillerat <akim.juillerat@camptocamp.com>
 * Thomas Nowicki <thomas.nowicki@camptocamp.com>
 * François Honoré <francois.honore@acsone.eu>
+* Hiep (Nguyen Hoang) <hiepnh@trobz.com>
```

Comment:
this is a straight migration, I don't think we add contribs here in such cases

Reply:
I keep it for now following the discussion here https://odoo-community.org/groups/contributors-15/contributors-200441?mode=thread&date_begin=&date_end=
I don't care to add myself but maybe that guy cares, better be too generous than the opposite IMHO

Reply:
yeah that's fine, I rolled back :wink: